### ISSUE TITLE
Add more PDF Files of standard books from different courses.

### ISSUE THREAD

**Issue by nitish-awasthi** [nitish-awasthi/Invertians-Archive#3]

**Comment by subhamagrawal7** [comment]
Any guidelines to be followed or should I directly make a pull request??

**Comment by Zenix27** [comment]
Hi @subhamagrawal7, 
We welcome any academic contribution, to any subject of Choice. Just keep in mind the way in which the folders are present, add the relevant file to the respective folder only. In the case of README contribution, we welcome any aesthetic changes/improvements.

**Comment by rkraghavkh18** [comment]
can i add ppt's related to chemical engineering but there is no chemical engineering directory.

**Comment by nitish-awasthi** [comment]
Yes @rkraghavkh18 ! You can add PPT/PDF,Notes of Chemical Engineering. I've created directory for the same in BTech/Chemical engineering.

**Comment by shristabaruah** [comment]
[Computer-Architecture-Organization.pdf](https://github.com/nitish-awasthi/Invertians-Archive/files/5395461/Computer-Architecture-Organization.pdf)
hi I am adding study material on computer architecture.Hope you like it.

**Comment by shristabaruah** [comment]
hi @nitish-awasthi can I add study material for computer architecture.

**Comment by nitish-awasthi** [comment]
Yes! you can.

**Comment by kartikagarwal9** [comment]
I wanted to contribute, can I go ahead and make a pull request ? @nitish-awasthi

**Comment by nitish-awasthi** [comment]
Hi @kartikagarwal9 ! You can file a PR.

**Comment by kartikagarwal9** [comment]
@nitish-awasthi Filed a PR consider it

**Comment by Ayu10x** [comment]
Can I add Data Structures and Algorithms in Java Sixth Edition by Michael T. Goodrich
